arangod/RestHandler/RestStatusHandler.cpp

```diff
@@ -145,11 +145,12 @@ RestStatus RestStatusHandler::executeStandard(ServerSecurityFeature& security) {
 
     if (!isStartup && !serverState->isSingleServer()) {
       result.add("persistedId", VPackValue(serverState->getPersistedId()));
-      if (auto rid = serverState->getRebootId(); rid.initialized()) {
-        result.add("rebootId", VPackValue(rid.value()));
-      }
 
       if (!serverState->isAgent()) {
+        // agents do not initialize the reboot id
+        if (auto rid = serverState->getRebootId(); rid.initialized()) {
+          result.add("rebootId", VPackValue(rid.value()));
```

Comment:
The problem here is, that `getRebootId` asserts that the reboot id is initialized.